src/Microsoft.ML.Data/Transforms/ConcatTransform.cs

```diff
@@ -306,6 +306,10 @@ private ColumnInfo[] LoadLegacy(ModelLoadContext ctx)
             //          int: index of the alias
             //          int: string id of the alias
             //      int: -1, marks the end of the list
+            // int: sizeof(Float).
+
+            var sizeofFloat = ctx.Reader.ReadInt32();
+            Contracts.CheckDecode(sizeofFloat == 4 || sizeofFloat == 8);
```

Comment:
>4 || sizeofFloat == 8 [](start = 49, length = 21)

`sizeof(float)` and `sizeof(double)` ?

Reply:
Actually, how about we disallow the double build? It's not like anyone was using it or something

---
In reply to: [226423915](https://github.com/dotnet/machinelearning/pull/1301#discussion_r226423915) [](ancestors = 226423915)